phpunit.xml.dist

```diff
@@ -5,8 +5,7 @@
         colors="true"
         displayDetailsOnTestsThatTriggerDeprecations="true"
         cacheDirectory="tmp/phpunit"
```

Comment:
```suggestion
        cacheResult="false"
```

tbf we might want to disable it instead, I've enabled it because we do keep `tmp/phpunit` directory and to resolve phpunit reporting the configuration as outdated due to `cacheResultFile` option being present

Reply:
Actually we are not; I've had some old artefacts locally 😎